libs/core/src/components/pds-table/stories/pds-table.stories.js

```diff
@@ -0,0 +1,227 @@
+import { html } from 'lit-html';
+import { extractArgTypes } from '@pxtrn/storybook-addon-docs-stencil';
+import { withActions } from '@storybook/addon-actions/decorator';
+
+export default {
+  argTypes: extractArgTypes('pds-table'),
+  component: 'pds-table',
+  decorators: [withActions],
+  parameters: {
+    actions: {
+      handles: ['onclick', 'pdsTableRowSelected'],
+    },
+  },
+  title: 'components/Table',
+};
+
+const BaseTemplate = (args) => html`
+<pds-table
+  compact="${args.compact}"
+  component-id="${args.componentId}"
+  fixed-column="${args.fixedColumn}"
+  responsive="${args.responsive}"
+  selectable="${args.selectable}"
+>
+  <pds-table-head>
+    <pds-table-head-cell class="${args.compact ? 'hydrated is-compact' : 'hydrated'}">Column Title</pds-table-head-cell>
+    <pds-table-head-cell class="${args.compact ? 'hydrated is-compact' : 'hydrated'}">Column Title</pds-table-head-cell>
```

Comment:
Isn't `hydrated` used and set by the stencil-core runtime? I don't think we need to have that a part of the Template

Reply:
This was an attempt to get the switches in storybook to work. The code works properly but the switch in the storybook controls does not. Should I remove and we can revisit the controls later?

Reply:
Yeah, we can revisit before merging into the base branch.